### ISSUE TITLE
Does not detect attributes on JSX intrinsic elements (ex, `button`, `div`)

### ISSUE THREAD

**Issue by wolever** [edvardchen/eslint-plugin-i18next#107]
With the default settings on version 6.0.1, attributes on JSX intrinsic elements (ex, `button`, `div`) are not detected.

For example:

```jsx
<button title="foo" /> – no error
<Button title="foo" /> – error (as expected)
```

Interestingly, this seems specific to valid elements; for example:

```jsx
<foo title="foo" /> – error (in addition to JSX compile error)
```

**Comment by edvardchen** [comment]
It's intentional😀since most of dom elements' attributes are invisible to users.
